src/Neo.Network.RpcClient/Models/RpcVersion.cs

```diff
@@ -103,13 +98,14 @@ private static string StripPrefix(string s, string prefix)
 
         public JObject ToJson()
         {
-            JObject json = new();
-            json["network"] = Protocol.Network; // Obsolete
-            json["tcpport"] = TcpPort;
-            json["nonce"] = Nonce;
-            json["useragent"] = UserAgent;
-            json["protocol"] = Protocol.ToJson();
-            return json;
+            return new()
+            {
+                ["network"] = Protocol.Network, // Obsolete
```

Comment:
not related to this pr, but @shargon why network is marked as obsolete, do you know this?

Reply:
Not sure